layouts/almalinux-day-germany-2024/single.html

```diff
@@ -339,14 +339,17 @@ <h2 style="font-size: 60px; letter-spacing: 0.80px; font-weight: 700; margin-bot
             <h3 style="font-size: 40px; letter-spacing: 0.80px; font-weight: 500; margin-bottom: 20px;">{{ i18n "March 16-17" }}</h3>
             <h4 style="font-size: 23px; letter-spacing: 0.80px; font-weight: 500; margin-bottom: 20px;">{{ i18n "CloudFest Hackathon" }}</h4>
             <span style="letter-spacing: 0.80px;">
-              {{ i18n "9:00 am" }}<br>
+              {{ i18n "10:00 am: Hackathon kicks off!" }}<br>
               <b>{{ i18n "Securing more infrastructure by" }}<br>
               {{ i18n "easing OS upgrades" }}</b>
             </span>
             <hr size=3px width=165px align="left" color="white" style="margin-top: 5px; margin-bottom: 5px;">
+            <img class="img-thumbnail" src="/landingpages/aldg-2024/CloudFesthackathon.jpg" style="object-fit: cover; max-width: 250px;">
```

Comment:
Can you add an `alt` text for this? While we're not super duper great about accessibility (YET!), adding `alt` to images is an excellent low-hanging fruit :).